### PR TITLE
Add principles draft doc

### PR DESCRIPTION
**Pull request by gordonbrander**
Draft principles doc. Fixes https://github.com/subconsciousnetwork/noosphere/issues/79.

@cdata @jsantell please TAL and offer feedback. Do these principles scan? Do any of them feel off? Is there anything we might refine or change?

My hope is to define a small set of enabling principles, without getting too prescriptive:

> The internet is a big tent. Noosphere hopes to be a big tent, too. We aim to keep this document focused on a small set of enabling principles. By maintining a minimal ideological footprint, we leave space for growth and pluralism.

### CODE REVIEW

File: design/principles.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+We aim to build protocols that provoke permissionless innovation—the ability to do new things without hitting coordination problems.
+
+### Noosphere aims for decentralization...
+
+> Evolution is the most decentralized thing that you can imagine. It is something that runs itself and is self-organizing at every level and at every scale. (Stewart Brand)
+
+Decentralization makes ecosystems resilient and enables many of the other values we embrace, including evolvability, pluralism, user-ownership, and subsidiarity.
+
+We value decentralization in protocol design, and avoid centralization. Centralization can be usefully defined as “the ability of a single entity or a small group of them to exclusively observe, capture, control, or extract rent from the operation or use of a function” ([IETF](https://www.ietf.org/archive/id/draft-nottingham-avoiding-internet-centralization-05.html), 2022).
```
Comment (jsantell) [inline design/principles.md:43]: Smart quotes used here `”`

File: design/principles.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+Noosphere aims to support three core kinds of exit at the protocol layer:
+
+- Own your data: Spheres act as personal data backpacks. Your data is yours and you can take it with you between app and services.
+- Own your identity: Noosphere's security is built on top of [self-sovereign](https://en.wikipedia.org/wiki/Self-sovereign_identity) authz primitives ([UCAN](https://ucan.xyz/)). You own and control your keys.
+- Self-sovereign social graph: like email, you can take your address book with you. You can change apps and services while keeping your followers and the people you follow.
+
+### Noosphere is evolvable
+
+> “I argue that much of the Internet's success can be attributed to its users' ability to shape the network to meet their own objectives.” (Abbate, 1999. Inventing the Internet)
```
Comment (jsantell) [inline design/principles.md:27]: smart quotes

File: design/principles.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+### Noosphere is for everybody
+
+> Plurality is the condition of human action because we are all the same, that is, human, in such a way that nobody is ever the same as anyone else who ever lived, lives, or will live. (Hannah Arendt)
+
+Pluralism is a core value for Noosphere. We want Noosphere to be accessible to anyone who wants to use it, and adaptable toward a plurality of uses. We value building technology that is accessible to everyone, regardless of gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other characteristics.
+
+### Design is navigating tradeoffs by values
+
+Design is the practice of navigating [wicked problems](https://en.wikipedia.org/wiki/Wicked_problem)—problems for which there is no one optimal solution. The values that guide Noosphere’s design aren’t absolutes, but they do frame a design space that is meaningful for the particular goals of this protocol.
```
Comment (jsantell) [inline design/principles.md:67]: Smart apostrophe (`’`) (is that what this is called??)

File: design/principles.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
+This document outlines values and motivations that have influenced the Noosphere protocol. We embrace growth and learning, so this document should be considered a living document.
+
+## Purpose
+
+Unstoppable tools for thinking together.
+
+## Principles
+
+These are some of the values that have become important to Noosphere’s design.
```
Comment (jsantell) [inline design/principles.md:9]: Smart apostrophe

**Review by jsantell — DISMISSED** [review]
Looks great @gordonbrander !

**Review by cdata — COMMENTED** [review]
I'll second @jsantell 's comments, but otherwise LGTM!

File: design/principles.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+This document outlines values and motivations that have influenced the Noosphere protocol. We embrace growth and learning, so this document should be considered a living document.
+
+## Purpose
+
+Unstoppable tools for thinking together.
```
[the remark this answers is not in the document]
Comment (gordonbrander) [reply design/principles.md]: Added

File: design/principles.md (hunk 1 above)
Comment (gordonbrander) [reply design/principles.md]: Fixed

**Review by cdata — APPROVED** [review]
This is awesome, thank you for putting in the work to make it public!

**Comment by jsantell** [comment]
Great updates on `Purpose` -- both concrete and open ended

**Comment by gordonbrander** [comment]
Closed in favor of https://github.com/subconsciousnetwork/noosphere/pull/83

**Pull request closed without merging**